openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -193,17 +151,12 @@ def process(self):
             # if no render layers are present, create default one with
             # asterisk selector
             if not layers:
-                render_layer = self._rs.createRenderLayer('Main')
+                render_layer = rs.createRenderLayer('Main')
                 collection = render_layer.createCollection("defaultCollection")
                 collection.getSelector().setPattern('*')
 
-            renderer = cmds.getAttr(
-                'defaultRenderGlobals.currentRenderer').lower()
-            # handle various renderman names
-            if renderer.startswith('renderman'):
-                renderer = 'renderman'
-
-            self._set_default_renderer_settings(renderer)
+            self.log.info("Applying default render settings..")
+            lib_rendersettings.RenderSettings.apply_defaults()
```

Comment:
Please put this behind a setting to be enabled/disabled. Especially with Arnold "resetting the full AOVs" list I'm already scared Artists will get very mad whenever Creating a Render instance will wreck their scenes. :D

Reply:
@Allan-I there is `apply_render_settings` defined in the settings schema but is unused (should be used probably here)